cartography/intel/gcp/util/common.py

```diff
@@ -5,7 +5,7 @@
 
 def parse_and_validate_gcp_requested_syncs(gcp_requested_syncs: str) -> List[str]:
     validated_resources: List[str] = []
-    for resource in gcp_requested_syncs:
+    for resource in gcp_requested_syncs.split(','):
```

Comment:
split on a `None` value would fail. We should handle this.

if we implement the default services list logic, then we will not have any issues.